src/angular-public/time-input/time-input/time-input.directive.ts

```diff
@@ -5,30 +5,54 @@ import {
   HostBinding,
   HostListener,
   Inject,
+  Input,
   Optional,
   Renderer2,
   Self,
 } from '@angular/core';
 import { NgControl } from '@angular/forms';
+import { InputDirective } from '@sbb-esta/angular-public/input';
 
 const REGEX_PATTERN = /[0-9]{3,4}/;
 const REGEX_GROUPS_WITH_COLON = /([0-9]{1,2})[.:,\-;_hH]?([0-9]{1,2})?/;
 const REGEX_GROUPS_WO_COLON = /([0-9]{1,2})([0-9]{2})/;
+const PLACEHOLDER_DEFAULT = 'HH:MM';
 
 @Directive({
   selector: 'input[sbbTimeInput]',
 })
 export class TimeInputDirective {
+  /**
+   * placeholder (defaults to HH:MM)
+   */
+  @Input()
+  @HostBinding('attr.placeholder')
+  get placeholder(): string {
+    return this._placeholder;
+  }
+  set placeholder(value: string) {
+    this._placeholder =
+      typeof value === 'undefined' || value === null ? PLACEHOLDER_DEFAULT : value;
```

Comment:
You could use the ?? operator here, which is equivalent to the undefined/null check.
```suggestion
    this._placeholder = value ?? PLACEHOLDER_DEFAULT;
```